.github/workflows/vissv2server-build.yml

```diff
@@ -0,0 +1,33 @@
+
+name: vissv2server build check
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    paths:
+    - ".github/workflows/vissv2server-build.yml"
```

Comment:
This indicates the branches/files that trigger a build for a PR. Maybe more should be added. As the build is quite quick one could alternatively remove the path criteria so that it build for all PRs.